File: InputOutput/somar.py
import time

start = time.perf_counter()
<<<<<<< HEAD
print(f"10 + 10 = {10+10}")
=======

a: int = 10
b: int = 10
print(f"A soma entre a:{a} e b:{b} é {a+b}")

>>>>>>> 25767b28e9e66d52afc2c4930ff9de1b8c3fddfa
end = time.perf_counter()

print(f"Tempo de execução {(end-start)} s")

File: InputOutput/Somar.java
public class Somar {
    public static void main(String[] args){

        long tempoInicial = System.currentTimeMillis();
        int a = 10;
        int b = 10;
        int soma = a + b;
        System.out.println("A soma entre a:10 e b:10 é " + soma);
        long tempoFinal = System.currentTimeMillis();
        System.out.printf("%f ms%n", (tempoFinal - tempoInicial) / 1000d);
    }
}